hatchery/config.go

```diff
@@ -89,19 +90,41 @@ type AllPayModels struct {
 	PayModels       []PayModel `json:"all_pay_models"`
 }
 
+type DbConfig struct {
+	DynamoDb dynamodbiface.DynamoDBAPI
+}
+
+type Gen3LicenseUserMap struct {
+	ItemId      string `json:"itemId"`
+	Environment string `json:"environment"`
```

Comment:
i am not sure if there is a golang convention about this, but i think it would be preferable to move these struct definitions out of `config.go` and into `gen3licenseusermaps.go` where they are being used